server/sonar-server/src/test/java/org/sonar/server/authentication/UserIdentityAuthenticatorTest.java

```diff
@@ -70,7 +69,6 @@
   @Rule
   public DbTester db = DbTester.create(system2);
 
-  private UserDbTester userTester = new UserDbTester(db);
```

Comment:
and having XXXTester instances provided by `DbTester` is a good idea, I was considering it to make it clearer the role of these classes (as I don't think their name is good enough at that job).